src/Proto3/Suite/DotProto/Parsing.hs

```diff
@@ -125,9 +126,71 @@ globalIdentifier = token $ string "." >> _identifier
 ----------------------------------------
 -- values
 
--- [issue] these string parsers are weak to \" and \000 octal codes
-stringLit :: ProtoParser String
-stringLit = stringLiteral <|> stringLiteral'
+strLit :: ProtoParser String
+strLit = doubleQuotedLiteral <|> singleQuotedLiteral
```

Comment:
To understand the correctness of this parser I would need to find the part of the proto3 specification that specifies the grammar for string literals look like in .proto files, but thus far I have been unable to locate that specification.  Do you know where that documentation resides?

Reply:
You can find the grammar here: https://developers.google.com/protocol-buffers/docs/reference/proto3-spec

The relevant part is:

```abnf
strLit = ( "'" { charValue } "'" ) |  ( '"' { charValue } '"' )
charValue = hexEscape | octEscape | charEscape | /[^\0\n\\]/
hexEscape = '\' ( "x" | "X" ) hexDigit hexDigit
octEscape = '\' octalDigit octalDigit octalDigit
charEscape = '\' ( "a" | "b" | "f" | "n" | "r" | "t" | "v" | '\' | "'" | '"' )
```

Reply:
Thank you very much!

Reply:
You're welcome!